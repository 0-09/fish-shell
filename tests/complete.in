logmsg Completion Wrappers

complete -c complete_test_alpha1 --no-files -a '(commandline)'
complete -c complete_test_alpha2 --no-files -w 'complete_test_alpha1 extra1'
complete -c complete_test_alpha3 --no-files -w 'complete_test_alpha2 extra2'

complete -C'complete_test_alpha1 arg1 '
complete -C'complete_test_alpha2 arg2 '
complete -C'complete_test_alpha3 arg3 '
