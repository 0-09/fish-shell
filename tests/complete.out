
####################
# Completion Wrappers
complete_test_alpha1 arg1 
complete_test_alpha1 extra1 arg2 
complete_test_alpha1 extra1 extra2 arg3 
